### ISSUE TITLE
Fix RPATH for CVMFS (ilc.desy.de) installation

### ISSUE THREAD

**Issue by shaojunlu** [iLCSoft/CED#4]
When the system directory "/usr/lib64" appear in the executable files "glced" RPATH, it will not be able to find the right "libstdc++.so.6" from CERN CVMFS compiler with "LD_LIBRARY_PATH" setup.

- Wrong one:
```
objdump -x ./bin/glced |grep RPATH
  RPATH                /cvmfs/ilc.desy.de/sw/x86_64_gcc49_sl6/CED/v01-09-02/lib:/usr/lib64
ldd  ./bin/glced
libstdc++.so.6 => /usr/lib64/libstdc++.so.6 (0x0000003862000000)
```

- "/usr/lib64" should be removed from RPATH list for the current CVMFS installation.
```
objdump -x ./bin/glced |grep RPATH
  RPATH                /cvmfs/ilc.desy.de/sw/x86_64_gcc49_sl6/CED/v01-09-02/lib
ldd  ./bin/glced
libstdc++.so.6 => /cvmfs/sft.cern.ch/lcg/contrib/gcc/4.9.3/x86_64-slc6/lib64/libstdc++.so.6 (0x00007fc4f4d05000)
```

- Could be fixed by this update in "CMakeLists.txt".
```
diff --git a/CMakeLists.txt b/CMakeLists.txt
index f1d4155..6928b32 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -86,7 +86,12 @@ IF( CED_SERVER )
     IF( NOT CED_NOT_INCLUDE_OPENGL_LINKER_PATH ) 
       FOREACH( _opengl_lib ${OPENGL_LIBRARIES} )
         GET_FILENAME_COMPONENT( _opengl_lib_path ${_opengl_lib} PATH )
-        LIST( APPEND _opengl_linker_paths ${_opengl_lib_path} )
+        # the RPATH to be used when installing, but only if it's not a system directory
+        LIST(FIND CMAKE_PLATFORM_IMPLICIT_LINK_DIRECTORIES ${_opengl_lib_path} isSystemDir)
+        #MESSAGE( "isSystemDir: ${isSystemDir}" )
+        IF("${isSystemDir}" STREQUAL "-1")
+          LIST( APPEND _opengl_linker_paths ${_opengl_lib_path} )
+        ENDIF("${isSystemDir}" STREQUAL "-1")
       ENDFOREACH()
 
     # remove duplicate paths
```

One access this "CMakeLists.txt" from here:
```
/cvmfs/ilc.desy.de/sw/x86_64_gcc49_sl6/CED/v01-09-02/CMakeLists.txt
```

**Comment by petricm** [comment]
For our installations we skip RPATH in CED since it cannot have influence on sim/reco. It might be worth considering removing RPATH form CED completely, but either way if fine.

**Comment by petricm** [comment]
Resolved with iLCSoft/CED#5
